ballerina/to_json_schema.bal

```diff
@@ -114,9 +112,9 @@ isolated function generateJsonSchema(string[] names, boolean[] required,
 
     if isArray {
         return <JsonArraySchema>{
-            \$schema: "https://json-schema.org/draft/2020-12/schema",
             items: schema,
-            'type: nilableType ? ["array", "null"] : "array"
+            'type: "array",
+            nullable: nilableType ? true: ()
```

Comment:
nullable isn't valid JSON schema, right? We shouldn't do this. We discussed changing to `oneOf`, right?

Reply:
It will be a major change, we are currently doing this until our record{} -> Json schema module available. Cant we stick to this temporaly?
cc @MohamedSabthar